helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -509,10 +510,15 @@ public void rollingUpdate(final DeploymentGroup deploymentGroup,
       final RollingUpdateOp op = getInitRollingUpdateOps(updated, MANUAL, client);
       operations.addAll(op.operations());
 
+      log.info("starting zookeeper transaction for rolling-update on "
```

Comment:
Rolling updates that are started when hosts change take a slightly different code path so those will not be logged with this change.